sdk_ctx.go

```diff
@@ -0,0 +1,32 @@
+// +build go1.7
+
+package equinox
+
+import (
+	"context"
+)
+
+// CheckContext is like Check but includes a context.
+func CheckContext(ctx context.Context, appID string, opts Options) (Response, error) {
+	var req, err = checkRequest(appID, &opts)
+
+	if err != nil {
+		return Response{}, err
+	}
+
+	req = req.WithContext(ctx)
+	return doCheckRequest(opts, req)
```

Comment:
Instead of this being two lines, can we just make this `doCheckRequest(opts, req.WithContext(ctx))`?